.travis.yml

```diff
@@ -27,7 +27,7 @@ before_install:
     - if [[ ${TRAVIS_OS_NAME} == "linux" ]] ; then export PATH=${HOME}/edm/bin:${PATH} ; fi
     - if [[ ${TRAVIS_OS_NAME} == "osx" ]]; then export PATH="${PATH}:/usr/local/bin" ; fi
     - edm install --version 3.6 -y click setuptools
-    - git clone git://github.com/force-h2020/force-bdss.git
+    - git clone git://github.com/force-h2020/force-bdss.git --branch enh/refactor-bdss-application
```

Comment:
Needed for testing, but remove before you merge.

Reply:
Now removed after merge of `force-bdss/enh/refactor-bdss-application` into master